assets/js/settings.js

```diff
@@ -4,9 +4,9 @@
 
     // storing RegExp strings for decoding the uploaded config file
     var RegExpStrings = {
-      channel_id: /channel_id: ([0-9a-zA-Z_-]+)/g,
-      key: /key: ([0-9a-zA-Z_-]+)/g,
-      secret: /secret: ([0-9a-zA-Z_-]+)/g
+      channel_id: /channel_id: ([^\s]+)/g,
```

Comment:
i'm probably interpreting this incorrectly, but is this just matching any single white space?

Reply:
nvm, matching up until..

Reply:
Yep, I think of it as "one or more non-whitespace characters"